contrib/gitlab.tpl

```diff
@@ -1,10 +1,11 @@
 {{- /* Template based on https://docs.gitlab.com/ee/user/application_security/container_scanning/#reports-json-format */ -}}
 {
-  "version": "2.3",
+  "version": "14.0.6",
   "vulnerabilities": [
   {{- $t_first := true }}
   {{- range . }}
   {{- $target := .Target }}
+    {{ $image := $target | regexFind "^[^:\\s]+" }}
```

Comment:
Remove the OS from the image.

While version `14.0.6` of the schema doesn't enforce the format, version `14.1.x` and later versions do. So I opted for using the string up to a space; e.g. `webgoat/webgoat-8.0:latest (debian 9.6)` becomes `webgoat/webgoat-8.0:latest`.

See also https://github.com/aquasecurity/trivy/issues/332.